ontology/yaml/resources/subfields/subfields.yaml

```diff
@@ -175,32 +211,45 @@ measurement:
   alarm: "Signal that an alarm is present."
   apparent: "The combination of reactive and real components (power)."
+  barometric: "Type of pressure used to measure the weight of atmosphere that surrounds." # Added 22/09/20
+  charging: "To put electricity into an electrical device such as a battery." # Added 05/10/20
   closed: "Indicates full closed (for two-position actuators)"
   deadband: "Represents a range in which the controller does not do anything."
+  delay: "a period of time by which something is late or postponed" # Added 27/08/20
   dewpoint: "The thermodynamic point at which water condenses from standing air."
   differential: "Measured difference between two reference points (e.g. differential_water_pressure)"
+  disable: "To stop something such as a machine or system from working." # Added 30/09/20
   efficiency: "The ratio of required input to actual output. "
   end: "Indicates a final part of something, especially a period of time."
-  fault : "Signal that a fault (low/medium) priority warning is present; can compromise system operation, but not necessarily imminently."
+  fault: "Signal that a fault (low/medium) priority warning is present; can compromise system operation, but not necessarily imminently."
+  flow: “Indication of moving water/air in a system” # added 25/08/20
   extent: "Distance travelled."
+  load: "The amount of work to be done by a device/machine, measured in percentage."
   match: "A function that matches something against others."
+  mismatch: "A function that represents a mismatch between commands and sensors/states." # Added 27/08/20
   offset: "The amount or distance by which something is out of line"
   open: "Indicates full open (for two-position actuators).  This is the default sense if unspecified"
   override: "An action that interrupts, cancels, or changes the current action or status."
   relative: "Quality of media with respect to theoretical minimum or maximum value for a given condition (e.g. relative humidity)."
   run: "State of being active or operating. By default applies to control program for system (eg VAV program run_command)."
   saturation: "Point at which no more of a material can be absorbed into another material."
+  shutdown: “Switching off or disabling of a system” # added 25/08/20
   speed: "Numeric setting of how fast to run a device, in the specified untis. Typically used to describe revolutions of a motor as a fraction of nominal or maximum."
   start: "Indicates a point in time or space at which something has its origin or beginning."
   static: "Resting or stagnant value (e.g. static_pressure_sensor)."
   thermal: "Relating to the transfer of heat."
   tilt: "Degree of radial rotation."
+  torque: "Force causing rotation of plant, which is measured in percentage." # Added 21/09/20
+  unbalance: "Represents voltage or current difference between phases and is meaured in percentage." 
```

Comment:
this term seems wonky.  do we have something more canonical?

Reply:
While "unbalance" is technically accurate, it could be more appropriate to use "phase angle" instead, and treat overall balance as a calculated field as an extension of the ontology.